Conceptpower+Spring/src/test/java/edu/asu/conceptpower/app/core/impl/RequestsManagerTest.java

```diff
@@ -19,59 +27,117 @@
+    public void test_getLatestReview_success() {
+        ReviewRequest response = requestManager.getLatestReview(CONCEPT_ID1);
         
-        ReviewRequest reviewRequest = new ReviewRequest();
-        reviewRequest.setStatus(ReviewStatus.OPENED); 
-        reviewRequest.setRequester("admin");
-        reviewRequest.setConceptId("WID-10126926-N-05-Einstein");
-        reviewRequest.setRequest("Testing Method2");
+        assertNotNull(response);
+        assertEquals(CONCEPT_ID1, response.getConceptId());
+        assertEquals(REVIEW_ID2, response.getId());
+        assertEquals("Review one more time", response.getRequest());
+        assertEquals("admin", response.getRequester());
+        assertEquals("admin", response.getResolver());
+        assertEquals(ReviewStatus.OPENED, response.getStatus());
         
-        Mockito.when(dbClient.getReviewRequestForConcept("WID-10126926-N-05-Einstein")).thenReturn(reviewRequest);
-
-        ReviewRequest fetchedReview = requestManager.getReview("WID-10126926-N-05-Einstein");
-     
-        Assert.assertEquals("WID-10126926-N-05-Einstein", fetchedReview.getConceptId());
-        Assert.assertEquals("admin", fetchedReview.getRequester());
-        Assert.assertEquals(ReviewStatus.OPENED, fetchedReview.getStatus());
-        Assert.assertEquals("Testing Method2", fetchedReview.getRequest());
     }
     
     @Test
-    public void test_getReview_failure() {
+    public void test_getLatestReview_failure() {
+        ReviewRequest fetchedReview = requestManager.getLatestReview("WID-10126926-N-05-Thomas");
         
-        ReviewRequest reviewRequest = new ReviewRequest();
-        reviewRequest.setStatus(ReviewStatus.OPENED); 
-        reviewRequest.setRequester("admin");
-        reviewRequest.setConceptId("WID-10126926-N-05-Einstein");
-        reviewRequest.setRequest("Testing Method2");
+        assertNull(fetchedReview);
+    }
+    
+    @Test
+    public void test_updateReview_success() {
+        Comment comment = new Comment();     
+        comment.setComment("Sample Comment");
         
-        Mockito.when(dbClient.getReviewRequestForConcept("WID-10126926-N-05-Einstein")).thenReturn(reviewRequest);
-
-        ReviewRequest fetchedReview = requestManager.getReview("WID-10126926-N-05-Thomas");
+        ReviewRequest response  = requestManager.updateReview(REVIEW_ID1, ReviewStatus.OPENED, comment, OffsetDateTime.now(), "admin");
+        
+        assertNotNull(response);
+        assertEquals(1, response.getComments().size());
+        assertEquals("Sample Comment",response.getComments().get(0).getComment());
+        assertEquals(CONCEPT_ID1, response.getConceptId());
+        assertEquals(REVIEW_ID1, response.getId());
+        assertEquals("Review the Spelling again", response.getRequest());
+        assertEquals("admin", response.getRequester());
+        assertEquals(ReviewStatus.OPENED, response.getStatus());
+    }
+    
+    @Test
+    public void test_updateReview_failure() {
```

Comment:
updating a review can fail for several reasons, you should test each. and the method name should reflect the failure reason.